showcase.json

```diff
@@ -2655,6 +2655,20 @@
     "submittableId": "32966927",
     "youtube": "https://www.youtube.com/watch?v=XTDznhHdeOE"
   },
+  {
+    "id": "survey_ease",
+    "name": "SurveyEase",
+    "oneLiner": "Redefining the Survey Landscape",
+    "website": "https://survey-ease.web.app",
+    "tags": [
+      "Tools / Infrastructure"
+    ],
+    "description": "A transformative survey experience, harnessing the power of emerging tech to provide a modern, user-friendly platform for efficient data gathering and insights.",
+    "stats": "Working MVP",
+    "logo": "/img/showcase/survey_ease_logo.png",
+    "usesInternetIdentity": true,
+    "twitter": "https://twitter.com/surveyEase"
+  },
```

Comment:
Can you please also add your Submittable ID?

Reply:
Done @jennifertrin